docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM julia:1.0.3-stretch
+
+# Sets up jupyterlab
+RUN apt-get update; \
+    apt-get install -y python3 python3-pip python3-dev build-essential git
+
+RUN pip3 install jupyter -U && pip3 install jupyterlab
```

Comment:
We don't need to install jupyter with pip because IJulia will install its own copy of a python environment.